### PR TITLE
No longer depend on a fork of materialize-css

### LINKED ISSUE #480: Dashboard icons
We're now using materialize css icon set (old version) http://materializecss.link/icons.html
They are now replaced by Google font: http://materializecss.com/icons.html

We should consider migrating to new version.

### PR DESCRIPTION
**Pull request by patrys**
One known bug: selects with `multiple` display the label of the first item when nothing is selected (upstream bug).

Fixes #480

### CODE REVIEW

**Comment by patrys** [comment]
Upstream pull request: https://github.com/Dogfalo/materialize/pull/3705

**Comment by patrys** [comment]
Initially I intended to also drop `select2` as a dependency but the multiselects offered by `materialize-css` are of rather crappy. I've thus reimplemented `select2` this time styling it to follow the material theme:

image

**Comment by patrys** [comment]
@car3oon @bogdal @mociepka looking for feedback! 🙏

File: package.json [hunk 1]
```diff
@@ -16,6 +16,25 @@
     "node": "5.10.x"
   },
   "dependencies": {
+    "bootstrap-sass": "^3.3.7",
```
Comment (mociepka) [inline package.json]: We are using caret and tilde in dependencies may be we should always use tilde? In my opinion automatic update to  the most recent major is not a good approach for dependencies. What is your opinion @patrys ?

Comment (patrys) [inline package.json]: We're using `npm shrinkwrap` which freezes all dependencies. All upgrades are manual.

**Pull request merged**